packages/test/test-end-to-end-tests/src/test/stashedOps.spec.ts

```diff
@@ -969,6 +970,75 @@ describeCompat("stashed ops", "NoCompat", (getTestObjectProvider, apis) => {
 		assert.strictEqual(map2.get(testKey), testValue);
 	});
 
+	it.skip("ensure single attach op sent in map", async function () {
```

Comment:
these don't belong in the stashed ops file. not sure there is a good place. you can alway just create a new file

Reply:
Maybe `packages\test\functional-tests\src\test\ddsHandleEncoding.spec.ts`?  You might be able to reuse or emulate the infra there as well (in line with the other comments in here)

Reply:
I don't think they can go in the handle encoding tests, since those use MockHandles (and I'm guessing that the issue wouldn't reproduce unless they're real handles)